XamlControlsGallery/ControlPages/SliderPage.xaml

```diff
@@ -27,21 +28,65 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        
         <local:ControlExample x:Name="Example2" HeaderText="A Slider with range and steps specified.">
-            <StackPanel x:Name="Control2" Orientation="Horizontal">
-                <Slider x:Name="Slider2" Header="Control header" Width="200" Minimum="500" Maximum="1000"
-                        LargeChange="100" StepFrequency="10" SmallChange="10" Value="800"/>
-            </StackPanel>
+            <local:ControlExample.Options>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto" MinWidth="80"/>
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock x:Name="MinimumLabel" Text="Minimum:" Margin="0,0,0,0" Grid.Row="0" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MinimumValue" Grid.Row="0" Grid.Column="1" Margin="10,-5,0,0" Value="500" SpinButtonPlacementMode="Compact" AutomationProperties.Name="{x:Bind MinimumValue.Name}"/>
+
+                    <TextBlock x:Name="MaximumLabel" Text="Maximum:" Margin="0,7,0,0" Grid.Row="1" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MaximumValue" Grid.Row="1" Grid.Column="1" Margin="10,5,0,0" Value="1000" SpinButtonPlacementMode="Compact" AutomationProperties.Name="{x:Bind MaximumValue.Name}"/>
+
+                    <TextBlock x:Name="StepFrequencyLabel" Text="StepFrequency:" Margin="0,5,0,0" Grid.Row="2" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="StepFrequencyValue" Grid.Row="2" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact" AutomationProperties.Name="{x:Bind StepFrequencyValue.Name}"/>
+
+                    <TextBlock x:Name="SmallChangeLabel" Text="SmallChange:" Margin="0,5,0,0" Grid.Row="3" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="SmallChangeValue" Grid.Row="3" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact" AutomationProperties.Name="{x:Bind SmallChangeValue.Name}"/>
+
+                    <TextBlock x:Name="LargeChangeLabel" Text="LargeChange:" Margin="0,5,0,0" Grid.Row="4" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="LargeChangeValue" Grid.Row="4" Grid.Column="1" Margin="10,5,0,0" Value="100" SpinButtonPlacementMode="Compact" AutomationProperties.Name="{x:Bind LargeChangeValue.Name}"/>
```

Comment:
You should set the `AutomationProperties.Name` property to something like "Minimum" instead of `{x:Bind MinimumValue.Name}`. Also you should set the `AutomationProperties.AccessibilityView` property to `Raw`. Sorry for the ping-pong on this PR 😅

Reply:
Learned something from `AutomationProperties`. Should be good now